### PR TITLE
Add notification button

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**

### CODE REVIEW

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java [hunk 1]
```diff
@@ -438,11 +439,14 @@ public void onClick(ClickEvent event) {
...
-	ToolStripButton cloudAutomationLinkButton = getCloudAutomationLinkButton();
-         cloudAutomationLinkButton.setIcon(Images.instance.pca_30().getSafeUri().asString());
-	 cloudAutomationLinkButton.setIconSize(25);
-	 cloudAutomationLinkButton.setBorder(GREY_BUTTON_BORDER);
-
+	    ToolStripButton cloudAutomationLinkButton = getCloudAutomationLinkButton();
+        cloudAutomationLinkButton.setIcon(Images.instance.pca_30().getSafeUri().asString());
+	    cloudAutomationLinkButton.setIconSize(25);
+	    cloudAutomationLinkButton.setBorder(GREY_BUTTON_BORDER);
+        ToolStripButton notificationPortalLinkButton = getNotificationPortalLinkButton();
```
Comment (lpellegr) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java]: As you mentioned during the Standup, this code could be factorized, at least in the method here.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java [hunk 2]
```diff
@@ -500,6 +506,18 @@ public void onClick(ClickEvent event) {
         return cloudAutomationButton;
     }
 
+    private ToolStripButton getNotificationPortalLinkButton() {
+        ToolStripButton notificationPortalButton = new ToolStripButton("Notifications");
+
+        notificationPortalButton.addClickHandler(new ClickHandler() {
+            public void onClick(ClickEvent event) {
+                Window.open("/notification-service", "", "");
```
Comment (lpellegr) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java]: Using _/notification-service_ assumes that all ProActive services are deployed on the same host. We should probably start thinking in the future about a configuration file that defines the location of the services.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java [hunk 3]
```diff
@@ -491,7 +497,7 @@ public void onClick(ClickEvent event) {
     private ToolStripButton getCloudAutomationLinkButton() {
         ToolStripButton cloudAutomationButton = new ToolStripButton("Cloud Automation");
 
-	cloudAutomationButton.addClickHandler(new ClickHandler() {
+        cloudAutomationButton.addClickHandler(new ClickHandler() {
             public void onClick(ClickEvent event) {
                 Window.open("/cloud-automation", "", "");
```
Comment (lpellegr) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java]: Second and third parameters to _Window.open_ seem optionals. Why to set them with an empty value?

**Comment by lpellegr** [comment]
LGTM.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java (hunk 2 above)
Comment (ShatalovYaroslav) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java]: I am absolutely agree we should start thinking about this in the future.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java (hunk 3 above)
Comment (ShatalovYaroslav) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java]: Unfortunately there is no 'open' method with one parameter in Window class. Only this method:
open(String url, String name, String features)

Comment (lpellegr) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java]: Ah yes, sorry, I was looking at the Javascript API :( 

http://www.w3schools.com/jsref/met_win_open.asp

**Comment by paraita** [comment]
LGTM

**Pull request merged**